website/docs/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview.md

```diff
@@ -1,12 +1,22 @@
 ---
-title: "Metadata API Overview"
+title: "Metadata API"
 id: "metadata-overview"
 ---
 
+## About the metadata API
+
 Every time that dbt Cloud runs a dbt project, it generates metadata which pertains to the accuracy, recency, configuration, and structure of the <Term id="view">views</Term> and tables in the warehouse. dbt Cloud serves a GraphQL API which supports arbitrary queries over this metadata; the endpoint for this API is `https://metadata.cloud.getdbt.com/graphql`. This API is an incredibly rich resource for evaluating data health longitudinally or at a point in time.
 
+## Prerequisites
+
+The metadata API is available to accounts on the _Team_ and _Enterprise_ plans, for any version >= dbt v0.19.0. Please note that artifacts generated with a version of dbt _less than_ v0.19.0 will not be accessible via the dbt Cloud metadata API.
```

Comment:
Is there a page we can link to that discusses how to upgrade a plan if they don't meet the pre-reqs?

Reply:
Added a link to the upgrade guides!